db/migrate/20150209230946_disable_mail_for_closed_account.rb

```diff
@@ -0,0 +1,15 @@
+class DisableMailForClosedAccount < ActiveRecord::Migration
+  def up
+  	closed_users = User.joins(:person).where(people: {closed_account: true})
+  	closed_users.each do |closed_user|
+  		closed_user.update_attribute(:disable_mail, true)
+  	end
```

Comment:
Let's use [update_all](http://api.rubyonrails.org/classes/ActiveRecord/Relation.html#method-i-update_all) on the relation here, that should be much faster.